code/datums/ert.dm

```diff
@@ -62,3 +62,12 @@
 	rename_team = "Janitor"
 	mission = "Clean up EVERYTHING."
 	polldesc = "a Nanotrasen Janitorial Response Team"
+
+/datum/ert/clown
+	roles = list(/datum/antagonist/ert/clown, /datum/antagonist/ert/clown/robust)
+	leader_role = /datum/antagonist/ert/clown/robust
+	teamsize = 7
+	opendoors = FALSE
+	rename_team = "HONK Squad"
+	mission = "Bring joy and happiness to all the crew the only way a clown can honk!"
```

Comment:
mission = "Bring joy and happiness to all the crew the only way a clown can. Honk!"

Reply:
Canned clowns :thumbsup:

Reply:
still not Honk